utils.py

```diff
@@ -0,0 +1,9 @@
+def separate_callback_data(data):
+    return data.split(";")
+
+
+def reformat_persian_date(date: str) -> str:
+    return date\
+        .replace('یکشنبه', 'یک‌شنبه')\
+        .replace('سه شنبه', 'سه‌شنبه')\
+        .replace('پنجشنبه', 'پنج‌شنبه')
```

Comment:
As I don't understand persian dates, a docstring explaining the purpose of this function would be great.

Reply:
This is so unnecessary and it's just for beautifying the output.
In fact with this function, i'm replacing a full space with half space which is a space with half size! This is sth in persian writing rules. I can add some explanation to this function to be more clear for others or just delete it.

Reply:
Any updates?!